x-pack/solutions/observability/plugins/observability/public/components/custom_threshold/custom_threshold_rule_expression.tsx

```diff
@@ -461,29 +551,24 @@ export default function Expressions(props: Props) {
         iconType="search"
         placeholder={placeHolder}
```

Comment:
I saw that for the es query rule we have a different placeholder based on the selected language, could it be helpful here as well?

Reply:
Good idea, will check it.

Reply:
Done in [f24a756](https://github.com/elastic/kibana/pull/229453/commits/f24a756e970e0319aebce8e6e2d09dfa59fce3cb)